wsdl/avvisatura/sac-common-types-1.0.xsd

```diff
@@ -120,7 +120,7 @@
 
     <xsd:complexType name="ctIdentificativoUnivocoPersonaFG">
         <xsd:sequence>
-            <xsd:element name="tipoIdentificativoUnivoco" type="stTipoIdentificativoUnivocoPersFG" minOccurs="1">
+            <xsd:element form="qualified" name="tipoIdentificativoUnivoco" type="stTipoIdentificativoUnivocoPersFG" minOccurs="1">
```

Comment:
why is this needed ?

Reply:
To fix the namespace problem on tipoIdentificativoUnivoco (the prefix sac: on the xml req for PagoPa)

Reply:
this makes the `soap` library generate the correct XML?

Reply:
I've tried with ```attributeFormDefault="qualified" ``` on ```sac-common-types-1.0.wsdl ```, but it did not fix the problem. The only way (for now) is to use the ```form="qualified" ``` for the cases afflicted.